db/functions/n1ql_function.go

```diff
@@ -53,7 +53,8 @@ func (fn *n1qlInvocation) Iterate() (sgbucket.QueryResultIterator, error) {
 	fn.n1qlArgs["user"] = &userArg
 
 	// Run the N1QL query:
-	iter, err := db.N1QLQueryWithStats(fn.ctx, fn.db.Bucket, db.QueryTypeUserFunctionPrefix+fn.name, fn.Code, fn.n1qlArgs,
+	// TODO: Multi-collection support for user functions is not implemented.
```

Comment:
For a user-defined query, what effect does the data store have?  I assumed multi-collection query support would work automatically if the query wasn't defining any keyspace tokens that needed to be replaced.  Is it just where stats are tracked?

Reply:
I don't know, haven't thought about this feature at all to try and figure out how it'd fit in with collections.